app/tests/test_views.py

```diff
@@ -56,6 +56,34 @@ def login(self, email=None, password=None):
         assert LOGGED_IN_SENTINEL in res.data
         return res
 
+class ActivityFeedTests(ViewTestCase):
+    def test_posting_activity_requires_full_name(self):
```

Comment:
Let's just leave this in, it's a better solution than spending more time dealing with this edge case.